scripts/zones/Oldton_Movalpolos/npcs/Scrawled_Writing.lua

```diff
@@ -6,19 +6,20 @@
 local ID = require("scripts/zones/Oldton_Movalpolos/IDs");
 require("scripts/globals/npc_util");
 -----------------------------------
+local scrawledWritingPositions = 
+{
+    [1] = {-16.806, 7.718, 14.155},
+    [2] = {-18.0, 12.0, -115.0},
+    [3] = {-150.0, 8.0, -252.0},
+}
 
-function onTrade(player,npc,trade)
-    local mob = GetMobByID(ID.mob.GOBLIN_WOLFMAN);
-    if ( npcUtil.tradeHas(trade, 4541) and not mob:isSpawned() ) then
-        player:confirmTrade();
-        local x = npc:getXPos();
-        local y = npc:getYPos();
-        local z = npc:getZPos();
-        mob:setSpawn(x-1,y,z)
-        SpawnMob(ID.mob.GOBLIN_WOLFMAN):updateClaim(player);
-        npc:setStatus(dsp.status.DISAPPEAR);
+function onTrade(player, npc, trade)
+    if npcUtil.tradeHas(trade, 4541) and npcUtil.popFromQM(player, npc, ID.mob.GOBLIN_WOLFMAN, {radius = 2, hide = 10}) then
```

Comment:
i assume this hide = 10 was in there for testing

Reply:
Yeah, changed it back to 900.